examples/wifi-echo/server/esp32/main/gen/af-gen-event.h

```diff
@@ -47,10 +47,8 @@
     extern void emberAfPluginReportingTickEventHandler(void);                                                                      \
     static void clusterTickWrapper(EmberEventControl * control, EmberAfTickFunction callback, uint8_t endpoint)                    \
     {                                                                                                                              \
-        emberAfPushEndpointNetworkIndex(endpoint);                                                                                 \
```

Comment:
Could you please comment those network index bits out, not remove them, so we can decide what to do with them when we deal with the relevant issue?